Sources/Brave/Frontend/Browser/Tab.swift

```diff
@@ -335,10 +330,15 @@ class Tab: NSObject {
       webView.scrollView.layer.masksToBounds = false
       webView.navigationDelegate = navigationDelegate
 
-      restore(webView, restorationData: self.sessionData?.savedTabData)
+      restore(webView, restorationData: self.sessionData)
 
       self.webView = webView
       self.webView?.addObserver(self, forKeyPath: KVOConstants.URL.rawValue, options: .new, context: nil)
+      self.webViewScrollViewObserver = webView.scrollView.observe(\.contentOffset, options: [.new]) { [weak self] _, change in
+        guard let self = self, let value = change.newValue else { return }
+        self.onWebViewScrolled?(self, value)
```

Comment:
so we don't get it for free after all? saving the scroll position
i wonder if its worth it now

Reply:
Removed. This was only needed for when Xcode force terminates an app, so it doesn't terminate "cleanly" (willTerminate, willResignActive, willEnterBackground not called).